environment.yml

```diff
@@ -0,0 +1,23 @@
+# Installs dependencies for YOLACT managed by Anaconda. 
+# TODO: you must additionally install nVidia drivers and CUDA, eg. on linux 
+# `apt install cuda-10-0` from https://developer.nvidia.com/cuda-10.0-download-archive 
+name: yolact-env
+prefix: /home/mmm/.conda/envs/yolact-env
+channels:
+        #  - pytorch
+  - defaults
+dependencies:
+  - python==3.7
+  - pip
+  - cython
+  - torchvision
+  - pytorch-gpu>=1.0 # you could also use pytorch (without -gpu) if you want to run on CPU and don't have an nVidia card
```

Comment:
The minimum pytorch version for YOLACT is 1.0.1, since 1.0 didn't have a piece of functionality that I needed from 1.0.1. Would it be sufficient to simply change this to `pytorch-gpu>=1.0.1`? I'm not sure how anaconda versioning works.

Reply:
yes, `pytorch-gpu >=1.0.1` (with the space). Fixed, thanks!